ios/Classes/PTFlutterViewController.m

```diff
@@ -353,10 +314,34 @@ - (void)applyNavIcon
     }
 }
 
-- (void)setThumbnailViewEditingEnabled:(BOOL)thumbnailViewEditingEnabled
+- (void)setThumbnailEditingEnabled:(BOOL)thumbnailEditingEnabled
+{
+    self.thumbnailsViewController.editingEnabled = thumbnailEditingEnabled;
+}
+
+- (BOOL)isThumbnailEditingEnabled
+{
+    return self.thumbnailsViewController.editingEnabled;
+}
+
+- (void)setAnnotationAuthor:(NSString *)annotationAuthor
+{
+    self.toolManager.annotationAuthor = [annotationAuthor copy];
+}
+
+- (NSString *)getAnnotationAuthor
+{
+    return [self.toolManager.annotationAuthor copy];
```

Comment:
Explicit call to copy not necessary.

Reply:
Or leave it to be extra safe in case `self.toolManager.annotationAuthor` isn't copy one day (which will never happen, but...)